tests/mocha/test.js

```diff
@@ -21,6 +21,14 @@ describe('Mocha smoke test', () => {
         $('elem').click()
     })
 
+    it('should chain properly', () => {
+        browser.isExistingScenario()
+
+        const el = browser.$('body')
+        el.$('.selector-1').isExisting()
+        el.$('.selector-2').isExisting()
```

Comment:
can we also assert the value to ensure that the promises got resolved?

Reply:
before the fix `el.$('.selector-2')` was Promise, so calling isExisting() was not working.
Anyway it makes sense to add additional verification